index.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+       <meta charset="UTF-8" />
+       <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+       <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+       <link rel="stylesheet" href="style.css" />
+       <title>My Mobile Portfolio</title>
+    </head>
+    <body>
+        <header>
+                <div class=""></div>
+                <div class=""></div>
```

Comment:
- [ ] Please remove these div tags. Please also remove any code that has no significance to the project.